multivalue_form_element.info.yml

```diff
@@ -1,4 +1,4 @@
-name: 'Multi-value form element'
-description: 'Provides a multi-value form element that wraps any number of form elements.'
+name: Multi-value form element
+description: Provides a multi-value form element that wraps any number of form elements.
 type: module
 core_version_requirement: ^8 || ^9
```

Comment:
Since we are now putting compatibility to ^8.9 || ^9.1 in composer.json, we can reflect that here.